### ISSUE TITLE
Presentation slides for all-hands (govt-wide accessibility report)

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3230]

**Comment by eelzi-ONRR** [comment]
Slides are nearly finished and a draft was sent to Nathan to help us get approval to present at the all-hands. Alex will make any final adjustments next week.

**Comment by AlexandracmONRR** [comment]
I changed the location of these slides to be in the FY24 folder. Please see the new location [in this link](https://doimspp.sharepoint.com/:f:/r/sites/NRRD/Shared%20Documents/General/Outreach/Outreach%20Proposals%20%26%20Presentations/FY24/508Assessment-AllHandsMeeting?csf=1&web=1&e=UUbY4b).

**Comment by AlexandracmONRR** [comment]
I ended up deleting my slide with the 4 quadrants and sticking with the matrix that was pulled from the report. To the note I just added the cell value meaning and also the definitions of maturity and conformance. Other than that I reviewed the other slides and didn't have changes - I think they contain detailed, onrr-relevant information pulled from the report that will be useful to present in the meeting.
